src/starbase/integration.ts

```diff
@@ -2,7 +2,9 @@ import { StarbaseConfig, StarbaseIntegration } from './types';
 import { executeWithLogging } from './process';
 
 async function collectIntegrationData(integrationDirectory: string) {
-  await executeWithLogging(`yarn --cwd ${integrationDirectory} start;`);
+  await executeWithLogging(
+    `yarn --cwd ${integrationDirectory} start --disable-schema-validation;`,
```

Comment:
I wonder if we should allow arbitrary args to be passed down to this command.

Reply:
I think we should allow that. I think, but am not completely sure, that invalid flags are ignored by the SDK.

Reply:
I believe the SDK will complain about an invalid option if you provide a flag it's not expecting.  I think either option is great and really appreciate the addition!

An argument in favor of allowing arbitrary args is it would also allow Starbase users to specify only a subset of steps to execute with the --step option.  And certainly other benefits as well, but that's the one that immediately comes to mind as potentially useful.